docs/manual/called-methods-checker.tex

```diff
@@ -0,0 +1,147 @@
+\htmlhr
+\chapterAndLabel{Called Methods Checker for the builder pattern}{called-methods-checker}
```

Comment:
Can we rewrite this material to indicate that checking the builder pattern is one use case for the called methods checker?  I'd like to bring out that you can use this checker anytime you need to check a property "Always call `a()` before `b()`"

Reply:
In particular I think the title of the chapter should just be "Called Methods Checker"